hamster/utils.py

```diff
@@ -96,3 +97,11 @@ def update_properties(properties):
 
     with open(properties_file_path, 'w') as config_file:
         config_parser.write(config_file)
+
+
+def get_telemetry_config():
+    """
+    Returns the telemetry configuration.
+    """
+    config_parser.read(properties_file_path)
+    return config_parser.getboolean('TELEMETRY', 'enabled')
```

Comment:
This will raise a ValueError if the value do not match the allowed list of values. You might want to add an exception handling
https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.getboolean